lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/RootContextLauncher.cs

```diff
@@ -31,16 +32,15 @@ namespace Org.Apache.REEF.Common.Runtime.Evaluator.Context
     /// </summary>
     internal sealed class RootContextLauncher
     {
-        private static readonly Logger LOGGER = Logger.GetLogger(typeof(RootContextLauncher));
-        
-        private readonly IInjector _rootServiceInjector = null;
+        private static readonly Logger Logger = Logger.GetLogger(typeof(RootContextLauncher));
 
+        private readonly IInjector _rootServiceInjector;
         private readonly IConfiguration _rootContextConfiguration;
-
-        private ContextRuntime _rootContext = null;
+        private ISet<object> _services;
```

Comment:
This is primarily for defensive programming such that C# always instantiates the variable upon assigning it.